guides/source/association_basics.md

```diff
@@ -1520,8 +1520,8 @@ The `collection.find` method finds objects within the collection. It uses the sa
 The `collection.where` method finds objects within the collection based on the conditions supplied but the objects are loaded lazily meaning that the database is queried only when the object(s) are accessed.
 
 ```ruby
-@open_orders = @customer.orders.where(open: true) # No query yet
-@open_order = @open_orders.first # Now the database will be queried
+@open_books = @author.books.where(open: true) # No query yet
```

Comment:
I'm not sure where the `open` attribute came from, or if it still applies to this collection.